### ISSUE TITLE
pre-train weights

### ISSUE THREAD

**Issue by fyf12138** [fourson/DeblurGAN-pytorch#9]
Can you provide a loss curve corresponding to the pre-trained weights on the GoPro dataset? Thank you！

**Comment by fourson** [comment]
Sorry, please train it yourself. I can only provide the pre-trained weights without the loss curve.
